apps/designer/app/designer/features/props-panel/control.tsx

```diff
@@ -174,13 +174,15 @@ type ControlProps = {
   component: Instance["component"];
   userProp: UserProp;
   onChangePropValue: (value: string | boolean, asset?: Asset) => void;
+  onChangePropAsset: (asset: Asset) => void;
```

Comment:
why not accepting Asset as a value in onChangePropValue? I just noticed that onChangePropValue also got a second param asset, which is not a good design